src/progressive-video-uploader.ts

```diff
@@ -3,6 +3,7 @@ import { PromiseQueue } from "./promise-queue";
 
 export interface ProgressiveUploadCommonOptions {
     preventEmptyParts?: boolean;
+    videoName?: string
```

Comment:
I think this could be put in [CommonOptions](https://github.com/apivideo/api.video-typescript-uploader/blob/main/src/abstract-uploader.ts#L41) (superclass of options that are common to progressive & regular uploads)